hack/make/.go-autogen

```diff
@@ -8,11 +8,12 @@ cat > autogen/dockerversion/dockerversion.go <<EOF
 package dockerversion
 
 var (
-	GITCOMMIT string = "$GITCOMMIT"
-	VERSION   string = "$VERSION"
+	GITCOMMIT    string = "$GITCOMMIT"
+	VERSION      string = "$VERSION"
```

Comment:
Then the binary would report that it's an experimental build anywhere the `VERSION` string is used without any extra logic, and wouldn't mess up the `bundle/$(cat VERSION)/` contract like what we have now does.